api/v1alpha2/linodemachine_types.go

```diff
@@ -74,6 +74,10 @@ type LinodeMachineSpec struct {
 	// DataDisks is a map of any additional disks to add to an instance,
 	// The sum of these disks + the OSDisk must not be more than allowed on a linodes plan
 	DataDisks map[string]*InstanceDisk `json:"dataDisks,omitempty"`
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
+	// +kubebuilder:validation:Enum=enabled;disabled
+	// DiskEncryption determines if the disks of the instance should be encrypted.
+	DiskEncryption string `json:"diskEncryption,omitempty"`
```

Comment:
should we set a default?

Reply:
The platform default is to have LDE enabled so I think leaving this empty should result in LDE=true as the platform default